ml-proto/test/memory.wasm

```diff
@@ -26,6 +27,17 @@
   (module (memory 100 1000 (segment 0 "a") (segment 2 "b") (segment 1 "c")))
   "data segment not disjoint and ordered")
 
+;; Test alignment annotation rules
+(assertinvalid (module (func (loadu.2.i8 (const.i32 0)))) "alignment too big")
```

Comment:
Agreed that the design doc isn't expected to be complete, but this specific case was discussed when creating the design, and we agreed that alignment just had to be a positive power of 2, no limits. This spec PR contradicts the design and what we agreed on, so the design is now wrong.

In cases like this the discussion should occur in the design repo: it revisits a decision that was made in the design repo. What the point of agreeing on anything in the design repo otherwise? I'm not saying that the design is golden and shouldn't change (witness the issues I've filed recently), I'm saying that we want to avoid diverging slightly in every place we implement wasm.

It's great the the spec repo finds unforeseen problems with the design, but this one wasn't unforeseen!